system-tests/README.md

```diff
@@ -1,122 +1,260 @@
-    ./gradlew -DuseFsVault="true" :launcher:shadowJar
-    ```
+```powershell
+$Env:REGISTRATION_SERVICE_LAUNCHER_PATH="c:/RegistrationService/launcher"
+docker-compose -f system-tests/docker-compose.yml up --build
+```
 
-- Go to Registration service root folder. And execute
+Once completed, following services will start within their docker containers:
+- 3 `EDC Connectors`
+  - _consumer-us_
+  - _consumer-eu_
+  - _provider_ (which will also be seeded with initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json))
+- A `Registration Service`
+- A `HTTP Nginx Server` (to serve DIDs) 
+- An `Azurite` blob storage service 
+  
 
-    ```bash
-    ./gradlew :launcher:shadowJar
-    ```
+_Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts._
 
-- Start MVD using docker-compose.yml file.
+`EDC Connectors` need to be registered using `Registration Service` CLI client jar. After publishing `Registration Service` locally the client jar should be available under the `Registration Service` root project folder in _client-cli/build/libs_.
 
-    ```bash
-    export REGISTRATION_SERVICE_LAUNCHER_PATH=Registration service launcher path e.g. `/home/user/RegistrationService/launcher`.
-    docker-compose -f system-tests/docker-compose.yml up --build
-    ```
+```bash
+# Replace path according to your local set up
+export REGISTRATION_SERVICE_CLI_JAR_PATH=c:/RegistrationService/client-cli/build/libs/registration-service-cli.jar
 
-  for windows powershell
+# Register Participants
+./system-tests/resources/register-participants.sh
+```
 
-    ```powershell
-    $Env:REGISTRATION_SERVICE_LAUNCHER_PATH = "Registration service launcher path e.g. /home/user/RegistrationService/launcher"
-    docker-compose -f system-tests/docker-compose.yml up --build
-    ```
+_Note for Windows PowerShell, the following commands should be run the the `MVD` root project folder._
 
-- This will start three EDC Connectors, one Registration Service, one HTTP Nginx Server to serve DIDs, Azurite blob storage service and also will seed initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json).
+```powershell
+# Replace path according to your local set up
 
-- `newman` docker container will automatically stop after seeding initial data from postman scripts.
+$Env:REGISTRATION_SERVICE_CLI_JAR_PATH = "c:\RegistrationService\client-cli\build\libs\registration-service-cli.jar"
 
-- EDC Connectors needs to be registered using Registration Service CLI client jar. After publishing RegistrationService locally the client jar should be available under `RegistrationService-Root/client-cli/build/libs` folder.
+# Register Provider
+java -jar $Env:REGISTRATION_SERVICE_CLI_JAR_PATH -s="http://localhost:8184/api" participants add --request="{ \`"name\`": \`"provider\`", \`"supportedProtocols\`": [ \`"ids-multipart\`" ], \`"url\`": \`"http://provider:8282\`" }"
```

Comment:
note that @Izzzu is changing this, will need to be updated

Reply:
then it can be addressed in a follow up